ml-proto/host/lexer.mll

```diff
@@ -231,6 +231,8 @@ rule token = parse
     { CONVERT (floatop t Float32Op.ConvertSInt64 Float64Op.ConvertSInt64) }
   | (fxx as t)".convert_u/i64"
     { CONVERT (floatop t Float32Op.ConvertUInt64 Float64Op.ConvertUInt64) }
+  | (ixx as t)".select" { SELECT ( intop t Int32Op.Select Int64Op.Select) }
```

Comment:
Nit: can you move these to a more consistent place (i.e., not in the middle of the conversion operators)?